testing/docker/Dockerfile.go112

```diff
@@ -0,0 +1,5 @@
+FROM golang:1.12
+
+COPY bootstrap.sh /bootstrap.sh
```

Comment:
It may be worth defaulting to modules/proxy in the container:

```
# Always use modules.
ENV GO111MODULE=on \
    GOPROXY=https://proxy.golang.org
```

Just so arbitrary use of the containers behaves as expected in a consistent way.